tests/std/tests/P1423R3_char8_t_remediation/test.compile.pass.cpp

```diff
@@ -45,18 +48,25 @@ constexpr bool test() {
 }
 
 // Control cases
-STATIC_ASSERT(test<std::ostream, char>());
-STATIC_ASSERT(test<std::wostream, wchar_t>());
+STATIC_ASSERT(test_stream_insertion<std::ostream, char>());
+STATIC_ASSERT(test_stream_insertion<std::wostream, wchar_t>());
 
 #ifdef __cpp_char8_t
-STATIC_ASSERT(!test<std::ostream, char8_t>());
-STATIC_ASSERT(!test<std::wostream, char8_t>());
+STATIC_ASSERT(!test_stream_insertion<std::ostream, char8_t>());
+STATIC_ASSERT(!test_stream_insertion<std::wostream, char8_t>());
 #endif // __cpp_char8_t
 
 #ifdef _NATIVE_WCHAR_T_DEFINED
-STATIC_ASSERT(test<std::ostream, wchar_t>() == !_HAS_CXX20);
+STATIC_ASSERT(test_stream_insertion<std::ostream, wchar_t>() == !_HAS_CXX20);
 #endif // _NATIVE_WCHAR_T_DEFINED
-STATIC_ASSERT(test<std::ostream, char16_t>() == !_HAS_CXX20);
-STATIC_ASSERT(test<std::ostream, char32_t>() == !_HAS_CXX20);
-STATIC_ASSERT(test<std::wostream, char16_t>() == !_HAS_CXX20);
-STATIC_ASSERT(test<std::wostream, char32_t>() == !_HAS_CXX20);
+STATIC_ASSERT(test_stream_insertion<std::ostream, char16_t>() == !_HAS_CXX20);
+STATIC_ASSERT(test_stream_insertion<std::ostream, char32_t>() == !_HAS_CXX20);
+STATIC_ASSERT(test_stream_insertion<std::wostream, char16_t>() == !_HAS_CXX20);
+STATIC_ASSERT(test_stream_insertion<std::wostream, char32_t>() == !_HAS_CXX20);
+
+#ifdef __cpp_char8_t
+void test_u8path() {
+    (void) std::filesystem::u8path(u8"a");
+    (void) std::filesystem::u8path(std::basic_string_view{u8"a"});
+}

```

Comment:
For completeness, I think we should additionally test the `u8path(InputIterator first, InputIterator last)` overload. I'll validate and push a change.